h/static/scripts/annotator/guest.coffee

```diff
@@ -197,10 +209,14 @@ module.exports = class Guest extends Annotator
   _setupViewer: -> this
   _setupEditor: -> this
   _setupDocumentEvents: ->
-    $(document).bind({
```

Comment:
You're going to need to seek an explanation for this or hand it off to someone who can do the iOS debugging necessary to figure out what's going on. I'm not satisfied by explanations like this, that it seems to work one way but not some other way. When we don't understand why something happens we can't make testable assertions about how it should behave and we can't explain in code or comments to the next person why it's written the way it is. In short, the result is fragile and confusing code that is, from an engineering perspective, as good as broken.